docs/charter.md

```diff
@@ -46,25 +51,23 @@ The _CPANSec_ intends to be a forum for coordinating and assisting in resolving
 - Share and document best security practices to authors and users;
 - Raise awareness to security threats and mitigations in Perl applications that rely on the CPAN ecosystem;
 - Act as a trusted intermediator for reporting potential security vulnerabilities to distributions, including managing and responsible disclosure of embargoed security information for a reasonable amount of time pertaining non-core distributions on CPAN, and communication with package managers and distribution channels;
-  - ~~In what ways / For how long? (e.g. max 14 days?) /  explain!~~
-    - Rationale: Let's keep specific embargo durations out of the charter, so we don't have to re-publish the charter when this changes. (garu)
 - Provide a forum for discussing and coordinating around supply chain security, integrity, metadata and the communication of these, including but not limited to:
-  - SBOM and other ways for tracking and communicating software changes, including interaction with external tooling and sources of security advisories;
+  - SBOM and other ways for tracking and communicating software and metadata changes, including interaction with external tooling and sources of security advisories;
   - The Update Framework (TUF) integration;
   - Third-party vulnerability scanning support;
 
 
-
 ### In-scope
 
 The _CPANSec_ concerns itself with a limited domain:
 
-- Distributions published on CPAN, including dual-life core modules;
-- Supply chain security, including CPAN/MetaCPAN and PAUSE
-  - [Artifact Transparency Logs](https://transparency.dev) like sigstore or sigsum
-  - Tooling and infrastructure for establishing chain-of-trust
-  - Tooling for downstream verification of chain-of-trust
-  - Tooling for the creation and verification of SBOM objects commonly in use
+- Distributions published on CPAN, not including dual-life core modules;
```

Comment:
I have no objection to NOT including core dists, but if that's the case then I think it's important to log that either in the PR text (I couldn't find it mentioned but maybe I just missed it) or in the commit itself, pointing to the resolution. This way, whenever sometime in the future someone asks "why?" we all have a go-to answer.

(this of course should be standard for all CPANSec who/what/how/why decisions)

Reply:
I've updated the PR text.